### PR TITLE
✨ Article Comments

### LINKED ISSUE #307: [news] Comments on news
## Short description of the feature
Add comments on news to increase engagement

[x] Relates to frontend
[x] Relates to the backend
[x] Requires modification of GraphQL
[x] Requires modification of Database
[x] Requires creating a new service

### PR DESCRIPTION
**Pull request by oliverlevay**
Closes #307 

Demo can be seen here https://dsek-frontend.herokuapp.com/

### CODE REVIEW

File: backend/services/core/migrations/20221105163821_article_comments.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      .references('articles.id')
+      .onDelete('CASCADE')
+      .comment('The article id');
+    t.uuid('member_id')
+      .unsigned()
+      .notNullable()
+      .references('members.id')
+      .onDelete('CASCADE')
+      .comment('The member id');
+    t.comment('A relation table for likes on news articles');
```
Comment (LucasBoberg) [inline backend/services/core/migrations/20221105163821_article_comments.ts:18]: Should be comments on news article not likes

File: frontend/components/Link.tsx [hunk 2]
```diff
@@ -9,24 +9,27 @@ function Link({
...
   style?: React.CSSProperties;
+  color?: string,
 }) {
   return (
     <NextLink href={href || ''} locale={locale}>
       <MuiLink
         underline={underline}
         height="fit-content"
-        display="flex"
+        // display="flex"
```
Comment (LucasBoberg) [inline frontend/components/Link.tsx:27]: Remove comment

**Review by LucasBoberg — CHANGES_REQUESTED** [review]
Only two fixes required otherwise it looks good! :)

**Comment by oliverlevay** [comment]
Thanks for your review @LucasBoberg , excellent suggestions, fixed now.

**Review by LucasBoberg — CHANGES_REQUESTED** [review]
Ability to click on the text to like and comment would be nice. Otherwise looks good :)

**Pull request merged**